Spec/Tests/RestClientChannelTests.swift

```diff
@@ -1488,4 +1488,26 @@ class RestClientChannelTests: XCTestCase {
             }
         }
     }
+    
+    // ??
```

Comment:
Can we write a feature spec point number here?

Reply:
https://github.com/ably/ably-cocoa/pull/1420/commits/8d9917c10bb226641e00e3176351519749a36940